uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -463,7 +464,7 @@ public void testCreateZone_ZoneAlreadyExists() {
         String errorMessage = String.format("The zone name %s is already taken. Please use a different " +
                                             "zone name", orchestratorZoneRequest.getName());
         assertEquals(JsonUtils.writeValueAsString(
-            new OrchestratorErrorResponse(errorMessage)), getResponseAlreadyExist.getBody());
+            new OrchestratorZoneResponse(zoneName, null, errorMessage, OrchestratorState.PERMANENT_FAILURE.toString())), getResponseAlreadyExist.getBody());
```

Comment:
It's better to compare OrchestratorZoneResponse than string is case serialization is implemented differently between controller and test.

I changed that in the next commit.